api/users/serializers.py

```diff
@@ -11,6 +11,29 @@
 )
 from api.base.utils import absolute_reverse
 
+class PublicFilesSerializer(JSONAPISerializer):
```

Comment:
As documented [elsewhere](https://github.com/CenterForOpenScience/osf.io/pull/5942/files#r71531078), this class can probably go away. However, it may be better to subclass `FileSerializer` and force guid creation there instead of how it's currently done.